src/test/java/edu/hawaii/its/api/configuration/GroupingsTestConfiguration.java

```diff
@@ -52,6 +55,10 @@ private <T> T getWsResultTestData(String propertyName, Class<T> type) {
         return JsonUtil.asObject(json, type);
     }
 
+    public List<OotbActiveProfile> ootbActiveProfilesTestData() {
+        return JsonUtil.asList(JsonUtil.readJsonFileToString("ootb.active.user.profiles.json"), OotbActiveProfile.class);
```

Comment:
I think it would be preferred to have two new functions asListFromFile and asObjectFromFile instead of readJsonFileToString that will always be wrapped by a asList or asObject function.

Reply:
Yeah sounds good!

Reply:
yeap! sounds good